api/scripts/config.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+
+from sqlalchemy import MetaData, create_engine
+from sqlalchemy.orm import sessionmaker
+
+from mod.src.settings import SETTINGS
+
+file_path = Path.cwd().joinpath("mod").joinpath("fixtures").joinpath("test_data.json")
+
+if not file_path.exists():
+    print("File does not exist")
+    exit(1)
+
+engine = create_engine(SETTINGS.database_url, connect_args={"check_same_thread": False})
```

Comment:
Do we want to use this script on heroku as well? Then we should here too do what #108 suggested

Reply:
It may be a good idea to always import the engine from `database.py` because that should be configured correctly (also applies to the loading and dumping scripts I think. But that might be for another PR, after this one and #108 are through?

Reply:
puh... I mean, this one already touched all the files mentioned anyways, so I'd also be fine just adding it in here. But if you would prefer a separate PR, we could also do this